src/cms/templates/pages/page_tree_node.html

```diff
@@ -103,7 +103,7 @@
             {% endif %}
         {% endif %}
         {% if page_translation %}
-        <a href="{% url 'copy_short_url' page_id=page.id region_slug=region.slug language_code=language.code %}" title="{% trans 'Copy short link' %}" class="py-3 px-2">
+        <a href="#" onclick="short_url_to_clipboard('{{ page_translation.short_url }}');" title="{% trans 'Copy short link' %}" class="py-3 px-2">
```

Comment:
I think this needs to go away from the page list, because the user cannot directly choose the translation for the URL.

Reply:
Once we fix the language switcher, it will no longer be a problem to change e.g. to the english page tree and export the short url from there...

Reply:
Alright.